src/ql/ir/new_to_old.cc

```diff
@@ -801,18 +801,30 @@ NewToOldConverter::NewToOldConverter(const Ref &ir) : ir(ir) {
     // would already have happened to the raw JSON data associated with
     // ir->platform.
     compat::PlatformRef old_platform;
+    QL_DOUT("NewToOldConverter");
     if (ir->platform->has_annotation<compat::PlatformRef>()) {
         old_platform = ir->platform->get_annotation<compat::PlatformRef>();
+        QL_DOUT("NewToOldConverter: got old_platform from annotation of new platform");
     } else {
         old_platform = compat::Platform::build(
             ir->platform->name,
             ir->platform->data.data
         );
+        QL_DOUT("NewToOldConverter: got old_platform by building it (compat::Platform::build) from new platform data");
     }
+    QL_DOUT("NewToOldConvertor old instruction_map:");
+    QL_DOUT("(disabled)");
+//    QL_IF_LOG_DEBUG {
```

Comment:
use #if/#else instead of comments to choose between disabled and active. Does this have generic use?